docs/modules/ROOT/pages/contributors/secret-rotation.adoc

```diff
@@ -14,12 +15,25 @@ Secrets should be rotated each twice a year (each 6 months):
 
 link:https://github.com/organizations/kubearchive/settings/secrets/actions[Organization Secrets]
 
-* OCI_PASSWORD: password for the robot account named after the variable OCI_USERNAME.
-Go to the
+OCI_PASSWORD::
+password for the robot account named after the variable OCI_USERNAME. Go to the
 link:https://quay.io/organization/kubearchive?tab=robots[KubeArchive's Quay Organization]
 and regenerate the token for the appropiate robot.
-* `KUBEARCHIVE_BOT_SECRET`: to be done.
-* `KUBEARCHIVE_RENOVATE_PRIVATE_KEY`: private key for the bot named
-`kubearchive-renovate`. Go to
+
+KUBEARCHIVE_BOT_SECRET::
+to be done.
+
+KUBEARCHIVE_RENOVATE_PRIVATE_KEY::
+private key for the bot named `kubearchive-renovate`. Go to
 link:https://github.com/organizations/kubearchive/settings/apps/kubearchive-renovate[the kubearchive-renovate app page]
 and generate a new client secret.
+
+== KubeArchive Repository
+
+`PUSH_TO_MAIN_DEPLOY_KEY`::
```

Comment:
To be consistent with the variables defined above, this should not have back-ticks around it. Or, the above variables should all have back-ticks around them (the file was inconsistent before you edited it).

```suggestion
PUSH_TO_MAIN_DEPLOY_KEY::
```

Reply:
Fixed